### PR TITLE
Feature/upgrade v4.1.1 add ignore exceptions

### PR DESCRIPTION
**Pull request by nikkonix**

### CODE REVIEW

File: manifests/agent/php.pp [hunk 1]
```diff
@@ -74,6 +74,7 @@
   $newrelic_daemon_proxy                                 = undef,
   $newrelic_daemon_collector_host                        = undef,
   $newrelic_daemon_auditlog                              = undef,
+  $newrelic_error_collector_ignore_exceptions            = 'Phprest\Exception\Exception',
```
Comment (Vincent--) [inline manifests/agent/php.pp]: Don't use a default value here, we can specify it when calling the class

File: manifests/php.pp [hunk 2]
```diff
@@ -70,6 +70,7 @@
   $newrelic_daemon_proxy                                 = undef,
   $newrelic_daemon_collector_host                        = undef,
   $newrelic_daemon_auditlog                              = undef,
+  $newrelic_error_collector_ignore_exceptions            = 'Phprest\Exception\Exception',
```
Comment (Vincent--) [inline manifests/php.pp]: Don't use a default value here, we can specify it when calling the class

File: manifests/agent/php.pp (hunk 1 above)
Comment (nikkonix) [reply manifests/agent/php.pp]: Fixed.

File: manifests/php.pp (hunk 2 above)
Comment (nikkonix) [reply manifests/php.pp]: Fixed.

**Review by Vincent-- — APPROVED** [review]

**Pull request merged**